src/lang/Language.php

```diff
@@ -23,115 +23,71 @@
 
 namespace pocketmine\lang;
 
-use function array_filter;
-use function array_map;
-use function explode;
-use function file_exists;
-use function is_dir;
 use function ord;
-use function parse_ini_file;
-use function scandir;
 use function str_replace;
 use function strlen;
 use function strpos;
-use function strtolower;
 use function substr;
-use const INI_SCANNER_RAW;
-use const SCANDIR_SORT_NONE;
 
-class Language{
-
-	public const FALLBACK_LANGUAGE = "eng";
+class Language
+{
+	/** @var string */
+	private $langCode;
+	/** @var string[] */
+	private $langValue;
```

Comment:
Nitpick: could be better named `$langValues` plural

Reply:
This is not applicable you can fix it